modules/game/client/controllers/game.client.controller.js

```diff
@@ -18,18 +18,29 @@ angular.module('game').controller('GameController', ['$scope', '$location', '$do
     $scope.CanvasSettings = CanvasSettings;
     $scope.ChatSettings = ChatSettings;
     $scope.GameSettings = GameSettings;
-    $scope.TopicSettings = TopicSettings;
     $scope.Object = Object;
 
+    // Get a list of topics from the server
+    $http.get('/api/topics').success(function (response) {
+      console.log('Retrieved list of topics', response);
+      $scope.listOfTopics = response;
+    }).error(function (response) {
+      console.error('Failed to fetch topics', response);
+    });
```

Comment:
Immediately fire off a request to grab the list of topics, so that the UI can be populated.

Note: UI binds to the `listOfTopics` variable.